doc/releases/migration-guide-3.6.rst

```diff
@@ -29,6 +29,15 @@ Build System
 Kernel
 ======
 
+* The system heap size and its availability is now determined by a ``K_MEM_POOL_HEAP_SIZE``
+  define instead of the :kconfig:option:`CONFIG_MEM_POOL_HEAP_SIZE` Kconfig option. Subsystems
+  can specify their own custom system heap size requirements by specifying Kconfig options with
+  the prefix ``CONFIG_MEM_POOL_HEAP_ADD_SIZE``. The old Kconfig option still exists, but will be
```

Comment:
first commit says HEAP_MEM_POOL_ADD_SIZE, this says MEM_POOL_HEAP_ADD_SIZE - both can't be right? Plus should have _ suffix

Reply:
> first commit says HEAP_MEM_POOL_ADD_SIZE, this says MEM_POOL_HEAP_ADD_SIZE - both can't be right? Plus should have _ suffix

Good catch. Now fixed.